docker/exec

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
what concerns me a bit of this command is that it mixes the meanings of docker-compose's exec and run. Do you see any problem renaming it to `docker/run` so we stick to that meaning?